container/Makefile

```diff
@@ -14,49 +14,50 @@ all: \
 
 # These are containers that will be built locally using the definition files in
 # the current folder
-local_containers := chewbbaca.sif \
-					postAlignQc.sif \
-					resfinder.sif \
-					virulencefinder.sif \
-					pythonScripts.sif
+local_containers := clinicalgenomicslund-bonsai-prp-0.3.1.img
 
 # These containers will be downloaded remotely
-remote_containers := abritamr.sif \
-					amrfinderplus.sif \
-					blast.sif \
-					bracken.sif \
-					bwakit.sif \
-					freebayes.sif \
-					kraken2.sif \
-					mlst.sif \
-					mykrobe.sif \
-					perl_json.sif \
-					quast.sif \
-					samtools.sif \
-					skesa.sif \
-					snippy.sif \
-					sourmash.sif \
-					spades.sif \
-					tbprofiler.sif
+remote_containers := depot.galaxyproject.org-singularity-ncbi-amrfinderplus-3.11.11--h6e70893_0.img \
+					depot.galaxyproject.org-singularity-bracken-2.8--py39hc16433a_0.img \
+					depot.galaxyproject.org-singularity-bwakit-0.7.17.dev1--hdfd78af_1.img \
+					depot.galaxyproject.org-singularity-chewbbaca-3.3.2--pyhdfd78af_0.img \
+					depot.galaxyproject.org-singularity-freebayes-1.3.6--hbfe0e7f_2.img \
+					depot.galaxyproject.org-singularity-kraken2-2.1.2--pl5321h9f5acd7_3.img \
+					depot.galaxyproject.org-singularity-mlst-2.23.0--hdfd78af_1.img \
+					depot.galaxyproject.org-singularity-mykrobe-0.12.2--py39h70e0db4_0.img \
+					depot.galaxyproject.org-singularity-perl-json-4.10--pl5321hdfd78af_0.img \
+					depot.galaxyproject.org-singularity-quast-5.2.0--py310pl5321hc8f18ef_2.img \
+					depot.galaxyproject.org-singularity-resfinder-4.4.2--pyhdfd78af_1.img \
+					depot.galaxyproject.org-singularity-samtools-1.17--h00cdaf9_0.img \
+					depot.galaxyproject.org-singularity-skesa-2.4.0--he1c1bb9_0.img \
+					depot.galaxyproject.org-singularity-snippy-4.6.0--hdfd78af_2.img \
+					depot.galaxyproject.org-singularity-sourmash-4.8.2--hdfd78af_0.img \
+					depot.galaxyproject.org-singularity-spades-3.15.5--h95f258a_1.img \
+					depot.galaxyproject.org-singularity-tb-profiler-5.0.1--pyhdfd78af_1.img \
+					depot.galaxyproject.org-singularity-virulencefinder-2.0.4--hdfd78af_1.img
+
+# URLs to Docker containers
+DOCKER_bonsai-prp := docker://clinicalgenomicslund/bonsai-prp:0.3.1
 
 # URLs to remote containers
-URL_abritamr := https://depot.galaxyproject.org/singularity/abritamr:1.0.14--pyhdfd78af_0
-URL_amrfinderplus := https://depot.galaxyproject.org/singularity/ncbi-amrfinderplus:3.11.11--h6e70893_0
-URL_blast := https://depot.galaxyproject.org/singularity/blast:2.14.0--hf3cf87c_0
```

Comment:
The Blast image is now missing, failing the build (See [this line](https://github.com/genomic-medicine-sweden/jasen/actions/runs/7855448654/job/21437187937#step:9:1412)).

Reply:
Ahh yes, I couldn't see that we needed the blast in the config so I thought it was an old error.